component/DashBoard/DashBoard.jsx

```diff
@@ -85,249 +66,216 @@ function DashBoard({ user }) {
   }
   async function fetchMonthlyLeaderBoardData() {
     await httpService
-      .post(`/api/dashboard/monthly_leaderboard`, {
+      .post(`/api/dashboard/leaderboard`, {
         date: currentMonth,
         userId: user.id,
       })
       .then(({ data: response }) => {
         if (response.status === 200) {
```

Comment:
why checking status if it is successful and getting in then?

Reply:
this is not a network status it is the custom status that we passing in the response to check its a success request